server/controllers/services/email.js

```diff
@@ -0,0 +1,175 @@
+/**
+ * Dependencies.
+ */
+const emailLib = require('../../lib/email');
+const Promise = require('bluebird');
+const config = require('config');
+const request = require('request-promise');
+const _ = require('lodash');
+
+/**
+ * Controller.
+ */
+module.exports = (app) => {
+
+  const models = app.set('models');
+  const errors = app.errors;
+
+  // TODO: move to emailLib.js
+  const sendEmailToList = (to, email) => {
+    const tokens = to.match(/(.+)@(.+)\.opencollective\.com/i);
+    const list = tokens[1];
+    const slug = tokens[2];
+    email.from = email.from || `${slug} collective <info@${slug}.opencollective.com>`;
```

Comment:
instead of `wwcodeaustin collective`, it's better to put the group's real name in there.